pretyping/pretyping.ml

```diff
@@ -1346,6 +1348,15 @@ let ise_pretype_gen flags env sigma lvar kind c =
   in
   process_inference_flags flags !!env sigma (sigma',c',c'_ty)
 
+let partial_flags = {
+  use_typeclasses = UseTCForConv;
+  solve_unification_constraints = false;
+  fail_evar = false;
+  expand_evars = true; (* questionable, TODO bench *)
```

Comment:
@ppedrot any clue if this is good or bad in terms of perf?

Reply:
I remember tweaking that some time ago, but I don't recall the consequences. Probably uneven results.